src/app/screens/ConfirmPayment/index.tsx

```diff
@@ -150,17 +150,12 @@ function ConfirmPayment(props: Props) {
                   payments until it is exhausted.
                 </p>
                 <div>
-                  <label
-                    htmlFor="budget"
-                    className="mb-1 block text-sm font-medium text-gray-700 dark:text-gray-400"
-                  >
-                    Budget
-                  </label>
-                  <CurrencyInput
+                  <TextField
```

Comment:
Component is called `TextField` but here it's used as a number field. Would `InputField` be a better name? Might be confusing for people thinking this should/will only be used for text or confuse it with a TextArea?

Reply:
Wasn't quite sure about the naming tbh, so I took inspiration from: https://mui.com/components/text-fields/

Reply:
Might want to call it just "<InputWithLabel />" at this point.

Reply:
In Material Design it's als called a Textfield, whether you enter number / string.
https://material.io/components/text-fields

Reply:
👍  Alright, I get that.  
Might be good to keep that in mind for future component naming. And document it i.e. "We stick to material-ui naming even though we don't use that framework" or something like that.

Reply:
Not sure we need that for all components but when naming becomes hard I like to take inspiration from proven ui libs.
But open to any name that be better :)

Reply:
No all good, the question was just popping up when I looked through it. Don't mind me :)

Reply:
No all good, the question was just popping up when I looked through it. Don't mind me :)